README.md

```diff
@@ -1 +1,29 @@
-# subspace-cli
\ No newline at end of file
+# Subspace-CLI
+is a great tool that heavily simplifies the terminal workflow of the farming process. \\
```

Comment:
is the `\\` intended?

Reply:
Also, I would intro with `Subspace-CLI is a great...`,

Reply:
`\\` is making a new line, yes it was intentional